Automattic/woocommerce-payments
# Add digital wallet settings fields

Fixes #1730 

#### Changes proposed in this Pull Request

This PR adds the digital wallet settings page for the options to style the 1-click checkout buttons like Apple Pay or Google Pay. The copy and titles were pulled from the latest design. Some of the design features are not an exact match, but got as close to the design without requiring major additions to existing WordPress components. The biggest differences I found:

- The ability to add additional styles to a radio button label. The component only accepts strings for the label property of the options. 
- The display of the preview area. I had begun digging more in to this, but was finding it could pose a challenge due to how these buttons are rendered. Created a [follow up issue](https://github.com/Automattic/woocommerce-payments/issues/1808) for the previews.



#### Testing instructions

1. Make sure [group settings feature flag](https://github.com/Automattic/woocommerce-payments/blob/54ef9057095d1d988db2d3f99da299273d1e5962/includes/class-wc-payments-features.php#L22) is enabled. 
2. Go to "wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments" and click "Customize appearance".
![image](https://user-images.githubusercontent.com/572862/117734140-10412f80-b1b0-11eb-8e9a-c71d9b63b2b7.png)
3. You should see a new page with URL "wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments_digital_wallets"
4. The page should have 3 sets of radio buttons for the 1-click checkouts theming options. The radio buttons should all be interactive.
5. Sidebar details and links should work and open in a new tab.

![Screen Shot 2021-05-17 at 9 51 42 AM](https://user-images.githubusercontent.com/68524302/118500275-a831a280-b6f5-11eb-979b-4cf197e5db07.png)


-------------------
